src/app/service/photos/photos.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PhotosService {
+
+  constructor(private http: HttpClient) { }
```

Comment:
Before you can inject `HttpClient`, you need to import it where it is defined. If you type `HttpClient`, VSCode should automatically pick up what you're trying to declare and suggest you which one to import.

To fix this, simply add `import { HttpClient } from '@angular/common/http';` to the top of the file.

Reply:
Oh yeah! Agree